app/policies/organization_policy.rb

```diff
@@ -10,27 +10,28 @@ def new?
     true
   end
 
-  def show?
+  def create?
     true
   end
 
-  def edit?
-    record.users.include?(user)
+  def show?
+    true
   end
 
-  def create?
-    true
+  def edit?
+                  # super_team_member is a team_member
+    user.admin? || user.is_team_member_of(record) && user.is_super_team_member?
```

Comment:
Do any of the conditions need to be wrapped in parentheses? Or is it possible to replace the `user.is_team_member_of(record) && user.is_super_team_member?` with `user_is_super_team_member?` 🤔

Reply:
the think is that we have to check that the user has a super_team_member role and that it is part of an organization (record). Maybe we can think of a way to make it more clear and easy to understand (changing the schema and role management)